Error while running dev environment using docker

This is the error which I am getting when I try to run the app using docker and locally 
server_1  | [00:50:24] [nodemon] starting `node ./website/server/index.js`
server_1  | npm ERR! code ELIFECYCLE
server_1  | npm ERR! errno 1
server_1  | npm ERR! habitica@4.118.0 start: `gulp nodemon`
server_1  | npm ERR! Exit status 1
server_1  | npm ERR! 
server_1  | npm ERR! Failed at the habitica@4.118.0 start script.
server_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
server_1  | 
server_1  | npm ERR! A complete log of this run can be found in:
server_1  | npm ERR!     /root/.npm/_logs/2019-11-01T00_50_27_136Z-debug.log

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,3 +1,2 @@
 node_modules
 .git
-website

--- a/Dockerfile-Dev
+++ b/Dockerfile-Dev
@@ -1,5 +1,11 @@
 FROM node:12
-WORKDIR /code
-COPY package*.json /code/
-RUN npm install
+
+# Install global packages
 RUN npm install -g gulp-cli mocha
+
+# Copy Habitica code into container and install dependencies
+WORKDIR /usr/src/habitica
+COPY . /usr/src/habitica
+
+RUN npm install
+RUN npm run postinstall

